### PR TITLE
zh: Add config change example

### PR DESCRIPTION
**Pull request by Win-Man**
### What is changed, added or deleted? (Required)
Add an example for updating task config.

### Which TiDB Data Migration (DM) version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [x] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

**Comment by Win-Man** [comment]
@csuzhangxc @kissmydb PTAL

File: zh/task-configuration-file.md [hunk 1]
```diff
@@ -78,3 +78,17 @@ mysql-instances:
...
 本小节定义具体的数据同步子任务，DM 支持从单个或者多个上游 MySQL 实例同步数据到同一个下游数据库实例。
 配置项说明参见以上示例配置文件中 `mysql-instances` 的注释。
+
+## 修改任务配置
+
+在某些情况下需要更新任务配置内容，比如重置数据同步任务时设置了 `remove-meta` 为 `true` 和 `task-mode` 为 `all`，当数据同步任务重置完成后，需要更新任务配置文件中的 `remove-meta` 设置为 `false` ，防止下一次启动任务的时候重新同步任务。
+
+因为 DM 集群会持久化保存任务配置，所以修改任务配置需要通过 `stop-task`、`start-task` 将修改后的配置更新到 DM 集群中，如果直接修改任务配置文件，但是不重启任务，配置变更不会生效，DM 集群重启时仍然会读取之前保存的任务配置。
+
+这边以修改 `remove-meta` 配置为例，举例说明任务配置修改步骤：
```
Comment (toutdesuite) [inline zh/task-configuration-file.md:88]:
```suggestion
这里以修改 `remove-meta` 配置为例，举例说明任务配置修改步骤：
```

File: zh/task-configuration-file.md [hunk 2]
```diff
@@ -78,3 +78,17 @@ mysql-instances:
 
 本小节定义具体的数据同步子任务，DM 支持从单个或者多个上游 MySQL 实例同步数据到同一个下游数据库实例。
 配置项说明参见以上示例配置文件中 `mysql-instances` 的注释。
+
+## 修改任务配置
+
+在某些情况下需要更新任务配置内容，比如重置数据同步任务时设置了 `remove-meta` 为 `true` 和 `task-mode` 为 `all`，当数据同步任务重置完成后，需要更新任务配置文件中的 `remove-meta` 设置为 `false` ，防止下一次启动任务的时候重新同步任务。
```
Comment (toutdesuite) [inline zh/task-configuration-file.md:84]:
```suggestion
在某些情况下需要更新任务配置内容，比如重置数据同步任务时设置了 `remove-meta` 为 `true` 和 `task-mode` 为 `all`，当数据同步任务重置完成后，需要更新任务配置文件中的 `remove-meta` 设置为 `false`，防止下一次启动任务的时候重新同步任务。
```

**Comment by Win-Man** [comment]
@toutdesuite PTAL

**Pull request merged**